src/main/java/io/github/kawamuray/wasmtime/Linker.java

```diff
@@ -29,6 +34,22 @@ public void define(String moduleName, String name, Extern extern) {
         return Optional.ofNullable(nativeGet(store.innerPtr(), module, name));
     }
 
+    public <T> Map<String, Extern> getAll(Store<T> store, String module) {
+        Map<String, Extern> map = new HashMap<>();
+        for (String name : nativeExterns(store.innerPtr(), module)) {
+            try {
+                map.put(name, nativeGet(store.innerPtr(), module, name));
+            } catch(Exception e) {
+                // ignore native errors for externs of unimplemented type
```

Comment:
Can you log it at least, with enabling logger by `@Slf4j` annotation to avoid users getting confused.

Reply:
As a user, I think it would be more confusing if I started seeing logs about externs that I had no idea even existed.
 
I'm not sure I like the idea of logs being emitted at all by this library TBH. I would prefer if there were no dependencies but that is your choice I guess.

I will add a log message at DEBUG level.